ignite/cmd/scaffold_type.go

```diff
@@ -9,8 +11,10 @@ import (
 // NewScaffoldType returns a new command to scaffold a type.
 func NewScaffoldType() *cobra.Command {
 	c := &cobra.Command{
-		Use:     "type NAME [field]...",
+		Use:     "type NAME [field:type] ...",
 		Short:   "Type definition",
+		Long:    fmt.Sprintf("Type information\n%s\n", supportFieldTypes),
+		Example: "   ignite scaffold type todo-item priority:int desc:string tags:array.string done:bool",
```

Comment:
```suggestion
		Example: "  ignite scaffold type todo-item priority:int desc:string tags:array.string done:bool",
```
 its non align 
 
![Screenshot 2023-10-18 at 13 48 14](https://github.com/ignite/cli/assets/2406457/d0d084ef-4123-401e-9d2c-5d47891a8e07)

------- 

![Screenshot 2023-10-18 at 13 48 10](https://github.com/ignite/cli/assets/2406457/60f524b3-2dbc-4eb9-ab67-1b82a66fe7e4)